src/test/web/templates/freemarker/missing-message-template.ftl

```diff
@@ -0,0 +1,8 @@
+<html lang="en_US">
+<body>
+<div>
+  Yo, nice template.
+  <span>[@control.message key="testing" default="the default message"/]</span>
```

Comment:
Should we leave a comment here to describe why this exists for testing purposes?

Reply:
Done/rebased